presets/4.3/vtx/rush_sa2.1_vtx_tables.txt

```diff
@@ -0,0 +1,48 @@
+#$ TITLE: Rush Tiny Tank, Tank Racing, Tank Ultimate Plus/Mini/II, and Tank Solo VTX tables
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: VTX
+#$ STATUS: COMMUNITY
+#$ KEYWORDS:  vtx, vtx table, rush tank, rush, rushfpv, tank ultimate, tank solo, tiny tank
+#$ AUTHOR: Directory
+#$ DESCRIPTION: VTX tables for the Rush Tiny Tank, Rush Tank Racing, Rush Tank Ultimate Plus, Rush Tank Ultimate II, and the Rush Tank Solo.
+#$ DESCRIPTION: Select in the options which model your particular rush VTX is. If it is not listed below it may not work because these VTX tables are meant for SA2.1 only. Older Rush VTX's (like the original tank ultimate) use an older SmartAudio protocol such as v1.1. These tables are by default the 37CH version for the US. Select 48CH to unlock more frequencies.
+#$ DISCLAIMER:  Only select one VTX model. All previous VTX settings will be reset. Some frequencies in BAND E have been replaced for the non 48CH version.
+#$ INCLUDE_DISCLAIMER: misc/disclaimer/en/vtxtable.txt
+#$ DISCUSSION:  https://github.com/betaflight/firmware-presets/pull/78
+
+#$ INCLUDE: presets/4.3/vtx/defaults.txt
+vtxtable bands 5
+vtxtable channels 8
+vtxtable band 1 BAND_A   A CUSTOM  5865 5845 5825 5805 5785 5765 5745 5725
+vtxtable band 2 BAND_B   B CUSTOM  5733 5752 5771 5790 5809 5828 5847 5866
+vtxtable band 3 BAND_E   E CUSTOM  5705 5685 5665 5665 5885 5905 5905 5905
+vtxtable band 4 AIRWAVE  F CUSTOM  5740 5760 5780 5800 5820 5840 5860 5880
+vtxtable band 5 RACEBAND R CUSTOM  5658 5695 5732 5769 5806 5843 5880 5917
+vtxtable powerlevels 4
+
+#$ OPTION BEGIN: (UNCHECKED) 48CH
+vtxtable bands 6
+vtxtable band 3 BAND_E   E CUSTOM  5705 5685 5665 5645 5885 5905 5925 5945
+vtxtable band 6 LOWRACE  L CUSTOM  5362 5399 5436 5473 5510 5547 5584 5621
+# OPTION END
+
+#$ OPTION BEGIN: (CHECKED) Tank Ultimate Plus/Mini/II
+vtxtable powervalues 14 23 27 29
+vtxtable powerlabels 25 200 500 800
+# OPTION END
+
+#$ OPTION BEGIN: (UNCHECKED) Tiny Tank
+vtxtable powervalues 14 20 23 25
+vtxtable powerlabels 25 100 200 350
+# OPTION END
+
+#$ OPTION BEGIN: (UNCHECKED) Tank Racing
+vtxtable powervalues 14 17 23 27
+vtxtable powerlabels 25 50 200 500
+# OPTION END
+
+#$ OPTION BEGIN: (UNCHECKED) Tank Solo
```

Comment:
and here 

so these replace the single rush tables I made?

Reply:
yes they include all the models. all smart audio 2.1. also the docs in the readme are wrong then. i copied the example from that

Reply:
do you want to fix the readme too ?

Reply:
the readme and PR instructions say to only do 1 thing at a time for every pull so ill honor that. maybe another pull.